app/components/yieldfarm/ShowYieldFarmDetails.js

```diff
@@ -403,7 +403,9 @@ const ShowYieldFarmDetails = ({ content, wallet, refreshTokenStaked }) => {
     if (wallet.signer !== 'signer' && pId != 0) {
       try {
         const lpTokens = await masterChefContract();
-        await lpTokens.withdraw(pId, 0);
+        const withdraw = await lpTokens.withdraw(pId, 0);
+        const { confirmations, status } = await fetchTransactionData(withdraw);
+        callRefreshFarm(confirmations, status);
       } catch (error) {
         console.error(error);
```

Comment:
> Remove console.log

As you can see, my PR did not add this console statement to the code, this can be done on another refactor PR since this PR is not a refactor PR. Anyways I can still remove it since removing it will not cause issues like your other review.